cmd/ocm-backplane/console/console.go

```diff
@@ -979,24 +971,19 @@ func replaceConsoleURL(containerURL string, userProvidedURL string) (string, err
 	return u.String(), nil
 }
 
-type postTerminationAction func() error
+type postTerminateFunc func() error
 
 // keep the program running in frontend and wait for ctrl-c
-func (e *execActionOnTermStruct) execActionOnTerminationFunction(action postTerminationAction) error {
+func (e *execActionOnTermStruct) execActionOnTerminationFunction(action postTerminateFunc) error {
 	sigs := make(chan os.Signal, 1)
 	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
-	done := make(chan bool, 1)
 	sig := <-sigs
```

Comment:
This is where the code will block until a signal is received. All the other `ctx.Done()` and `close(done)` were not needed since they were all used in locations where the code was _not_ blocking. The exception being `exec.Command().Run()`, but those commands were `docker/podman run --detach` and return as soon as the container starts.

Reply:
Yeah .. This makes sense .. I hope this may not affect the [cleanup func](https://github.com/openshift/backplane-cli/blob/main/cmd/ocm-backplane/console/console.go#L701) where it deletes already existing containers ( if you log in to two or more cluster consoles )

Reply:
If you run this locally you'll see it cleans up containers on SIGINT (ctrl+c) on docker and podman now. The pre-flight cleanup of existing containers happens regardless of the channels and before the new containers were launched. Or were you referring to something else?

Reply:
I will test this locally and check how it works.. ( I may need some time ) 
I guess this may clean the containers for SIGINT anyway, but console containers time-out after some time ( ~30mins) . hope this will delete existing containers via pre-flight cleanup..

Reply:
Right that all works the same as it did before, you can see in the log output in the PR body that step running still as expected, issued here https://github.com/openshift/backplane-cli/blob/a272694db200f7ebb0fca34f2519e118c374df1d/cmd/ocm-backplane/console/console.go#L253

```
...
INFO[0004] Starting initial cleanup of containers
INFO[0004] Container monitoring-plugin-1ik2acojo5ckhbg9l7d4s6sjjbior7ce does not exist, no need to clean up
INFO[0004] Container console-1ik2acojo5ckhbg9l7d4s6sjjbior7ce does not exist, no need to clean up
```